packages/blade/src/components/Amount/__tests__/Amount.native.test.tsx

```diff
@@ -115,5 +115,15 @@ describe('<Amount />', () => {
     expect(formatAmountWithSuffix({ value: 10000000, currency: 'MYR', suffix: 'none' })).toBe(
       '10,000,000',
     );
+    // Related issue - https://github.com/razorpay/blade/issues/1572
+    expect(formatAmountWithSuffix({ value: 2.07, currency: 'INR', suffix: 'decimals' })).toBe(
+      '2.07',
+    );
+    expect(formatAmountWithSuffix({ value: 2.077, currency: 'INR', suffix: 'decimals' })).toBe(
+      '2.08',
+    );
+    expect(formatAmountWithSuffix({ value: 2.3, currency: 'INR', suffix: 'decimals' })).toBe(
+      '2.30',
```

Comment:
We can add few extra random edge cases also. 

0.2, 0.3, 100.1, 101.15, 2000.25 etc